crates/router/src/core/payments/helpers.rs

```diff
@@ -4031,7 +4037,33 @@ where
                 }
             }
         }
-        Some(connector_data_list)
+
+        let fallback_connetors_list = crate::core::routing::helpers::get_merchant_default_config(
```

Comment:
Nit:
```suggestion
        let fallback_connectors_list = crate::core::routing::helpers::get_merchant_default_config(
```

Not a blocker